roles/acm_sno/tasks/pre-run.yml

```diff
@@ -21,39 +21,6 @@
     msg: "CRDs are not present"
   when: acm_crd.resources | list | count == 0
 
-- name: "Get Storage Classes"
```

Comment:
Maybe checking `AgentServiceConfig` is defined, otherwise fail that requirement?